services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java

```diff
@@ -27,15 +27,15 @@
   private static final Optional < String > host;
   private static final Optional < String > port;
 
-  public static Integer i = new Integer(123);
+  //public static Integer i = new Integer(123);
   
   static {
     PROTOCOL = "http://";
     host = Optional.ofNullable(System.getenv("SERVICES_GJ_HOSTNAME"));
     port = Optional.ofNullable(System.getenv("SERVICES_GJ_PORT"));
     BASE_URI = PROTOCOL + host.orElse("localhost") + ":" + port.orElse("80") + "/";
     String uname = "steve";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "uname" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.89.108.85:9000/coding_rules#rule_key=squid%3AS1481)